homeassistant/components/russound_rio/quality_scale.yaml

```diff
@@ -0,0 +1,86 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling:
+    status: exempt
+    comment: |
+      This integration uses a push API. No polling required.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
I am missing a test where we try to add the same device twice by first adding a mock config entry and then trying to initiate the flow

Reply:
Will address in follow-up PR.